src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationRun.cs

```diff
@@ -220,29 +227,32 @@ private void updateRunResult(OptimizationRunResult currentResult)
       private IReadOnlyList<SimulationRunResults> runSimulations()
       {
          var simulationResults = new ConcurrentBag<SimulationRunResults>();
-         var parallelOptions = new ParallelOptions
-         {
-            CancellationToken = _cancellationToken,
-            MaxDegreeOfParallelism = _parameterIdentification.IsSingleRun ? _coreUserSettings.MaximumNumberOfCoresToUse : 1
```

Comment:
This was the actual problem here. We were only running on one core because we assume that we would achieve max // from the categorial variation (calculation method). Using Parallel.ForEach should not create too much parallelism